projects/addon-commerce/enums/currency-code.ts

```diff
@@ -1,21 +1,179 @@
+/**
+ * Цифровые коды валют с их описанием
```

Comment:
@vakhramoff Hi! Thanks for PR.
Our library is used not only by Russian developers but also by other languages' developers.

Please, use only English  in your PR (as international language).

Reply:
Okey, chief. I've done this.